sipn_reanalysis_ingest/luigitasks/main.py

```diff
@@ -23,11 +22,11 @@
 def make_daily_processing_requirement(
     *,
     date: dt.date,
-) -> Union[
-    Grib2InDailyAnd5DailyTarsToDailyNc,
-    Grib2In5DailyTarToDailyNc,
-    Grib2InDailyTarToDailyNc,
-]:
+) -> (
+    Grib2InDailyAnd5DailyTarsToDailyNc
+    | Grib2In5DailyTarToDailyNc
+    | Grib2InDailyTarToDailyNc
+):
```

Comment:
I think this is the coolest part about `ruff` vs. `flake8`. It can autofix almost everything, even more complicated-looking structures like this. Here it's automatically updating the code to the most modern Pythonic practices. @encassano you may really find this useful; instead of having to follow the advancements of Python practices, your code can automatically be re-written to follow them!